src/testIntegration/java/uk/gov/justice/digital/delius/controller/wiremock/DeliusApiMockServer.java

```diff
@@ -49,6 +49,39 @@ public void stubPostNsiToDeliusApi() {
         ));
     }
 
+
+    public void stubPatchNsiToDeliusApi() {
+        stubFor(patch(urlPathMatching("/v1/nsi/2500018596")).willReturn(aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withStatus(200)
+            .withBody("{\n" +
```

Comment:
in future we can use a multiline string for readability

Reply:
Done